packages/shared/src/components/filters/MyFeedHeading.tsx

```diff
@@ -34,21 +42,42 @@ function MyFeedHeading({
     return <FeedHeading>My feed</FeedHeading>;
   }
 
+  const alertProps: Omit<AlertPointerProps, 'children'> = {
+    offset: sidebarRendered ? [4, 0] : [-32, 4],
+    isAlertDisabled,
+    onClose: () => onUpdateAlerts({ myFeed: null }),
+    className: {
+      label: 'w-44',
+      message: classNames(
+        'bg-theme-bg-primary',
+        !sidebarRendered ? 'ml-4' : null,
+      ),
+      wrapper: 'mr-auto z-modal',
+    },
+    message: filterAlertMessage,
+    placement:
+      sidebarRendered && searchVersion === SearchExperiment.Control
+        ? AlertPlacement.Right
+        : AlertPlacement.Bottom,
+  };
+
+  if (searchVersion === SearchExperiment.V1) {
```

Comment:
We do render for both right?

Reply:
Yes, we have the default that starts at line 77.